OrbitAccessibility/include/OrbitAccessibility/AccessibleObjectFake.h

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_GL_ACCESSIBLE_OBJECT_FAKE_H_
+#define ORBIT_GL_ACCESSIBLE_OBJECT_FAKE_H_
```

Comment:
Adjust the include guards.

Reply:
Damn, I thought I double-checked all of them... apparently not. Thanks for spotting.